libraries/botbuilder-dialogs-adaptive/package.json

```diff
@@ -28,6 +28,12 @@
     }
   },
   "dependencies": {
+    "@microsoft/recognizers-text-sequence": "~1.1.4",
+    "@microsoft/recognizers-text": "~1.1.4",
+    "@microsoft/recognizers-text-number-with-unit": "~1.1.4",
+    "@microsoft/recognizers-text-number": "~1.1.4",
+    "@microsoft/recognizers-text-choice": "~1.1.4",
+    "@microsoft/recognizers-text-date-time": "~1.1.4",
```

Comment:
Why are we moving these?

Reply:
@mrivera-ms, moving these libraries from `devDependencies` to `dependencies` was required due to the rule [import/no-extraneous-dependencies](https://github.com/import-js/eslint-plugin-import/blob/v2.22.1/docs/rules/no-extraneous-dependencies.md).
The following image shows an example.
![imagen](https://user-images.githubusercontent.com/62260472/152351542-bff8f7be-d790-470b-8c14-9c1a069ad61b.png)